pkg/controller/backup_sync_controller.go

```diff
@@ -232,6 +238,30 @@ func (c *backupSyncController) run() {
 			default:
 				log.Debug("Synced backup into cluster")
 			}
+
+			// process the pod volume backups from object store, if any
+			podVolumeBackups, err := backupStore.GetPodVolumeBackups(backupName)
+			if err != nil {
+				log.WithError(errors.WithStack(err)).Error("Error getting pod volumes for this backup from backup store")
+				continue
+			}
+
+			for _, podvolumeBackup := range podVolumeBackups {
+				log = log.WithField("podVolumeBackup", podvolumeBackup.Name)
+				log.Debug("Checking this pod volume backup to see if it needs to be synced into the cluster")
+
```

Comment:
We'll need to update the PVB's owner ref's UID to match the backup from L230, since if we're syncing into a different cluster, it'll be different. It looks like we also set a label on the PVB with this UID, so we need to update that as well (see https://github.com/heptio/velero/blob/master/pkg/restic/backupper.go#L236 and https://github.com/heptio/velero/blob/master/pkg/restic/backupper.go#L242)